requirements.txt

```diff
@@ -1,24 +1,29 @@
 alembic==1.8.1
 ascii_magic==2.3.0
--e git+https://github.com/abrichr/atomacos.git; sys_platform == 'darwin'
+atomacos @ git+https://github.com/abrichr/atomacos.git#egg=atomacos ; sys_platform == 'darwin'
 bokeh==2.4.3
 clipboard==0.0.4
 deepdiff[optimize]==6.3.0
 dictalchemy3==1.0.0
+drawsvg==2.1.1
 fire==0.4.0
+gpt4all
 ipdb==0.13.11
 loguru==0.6.0
 matplotlib==3.6.2
 mss==6.1.0
+numpy==1.24.3
 openai==0.27.5
 pandas==2.0.0
+pillow==9.5.0
 pygetwindow==0.0.9; sys_platform == 'win32'
 pyinstaller
 setuptools-lint
 sphinx
 python-dotenv==1.0.0
 pywin32==306; sys_platform == 'win32'
 -e git+https://github.com/abrichr/pynput.git#egg=pynput
+pypotrace==0.3
```

Comment:
Is this still required?

Reply:
No. It has been deleted in [7d563f9](https://github.com/MLDSAI/OpenAdapt/pull/198/commits/7d563f99148ff1ab65d6b6b58c78c386fb4fab12)